src/libraries/System.IO.Pipelines/src/System/IO/Pipelines/PipeAwaitable.cs

```diff
@@ -19,7 +19,7 @@ internal struct PipeAwaitable
         private CancellationTokenRegistration _cancellationTokenRegistration;
 
 #if (!NETSTANDARD2_0 && !NETFRAMEWORK)
-        private CancellationToken CancellationToken => _cancellationTokenRegistration.Token;
+        private readonly CancellationToken CancellationToken => _cancellationTokenRegistration.Token;
 #else
         private CancellationToken _cancellationToken;
         private CancellationToken CancellationToken => _cancellationToken;
```

Comment:
If line 22 can be readonly can't 25 also be readonly? they're the same property in different builds.